lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+class Menu
+
+	DISHES = [
+		{:name => "Pizza", :price => 9.99},
+		{:name => "Kebab", :price => 4.99},
+		{:name => "Chicken Wings", :price => 3.49},
+		{:name => "Chips", :price => 2.99},
+		{:name => "Ice-cream", :price => 4.99}]
+
+	attr_reader :dishes, :dish_class
```

Comment:
why make `dish_class` an attribute?

You shouldn't expose `dishes` here either - it breaks encapsulation.  The `Menu` class should provide its own interface to query the dishes.